examples/sites/public/static/css/design-common.css

```
@@ -553,3 +553,268 @@
+    width: 100%;
+    text-align: left;
+  }
+  .tinyui-design-footer .footer-content .contact {
+    width: 100%;
+    text-align: center;
+    display: flex;
+    flex-direction: column-reverse;
+  }
+  .tinyui-design-footer .footer-content .contact .group-code {
+    margin-left: 0;
+    margin: 30px auto 12px;
```

Biome

[error] Unexpected shorthand property margin after margin-left



(lint/suspicious/noShorthandPropertyOverrides)